uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/PaymentMVP.kt

```diff
@@ -8,19 +8,21 @@ interface PaymentMVP {
 
     interface View {
 
-        fun bindCardDetails(savedPaymentInfo: SavedPaymentInfo?)
+        fun bindPaymentDetails(savedPaymentInfo: SavedPaymentInfo?, quotePrice: QuotePrice? = null)
```

Comment:
I think we dont need to assign null to quotePrice (it is optional variable)

Reply:
See above